src/components/layout.scss

```diff
@@ -11,6 +11,7 @@
 .content-container {
   max-width: var(--max-width-calc);
   padding: var(--contentContainerPadding);
+  width: 100%;
```

Comment:
Do we need this for some reason on the contact page? Since this is a class used globally, I'd prefer not to add this unless you are sure it isn't affecting other pages.

Reply:
Removing this causes the contact page to shrink in size but doesn't seem to affect any of the other pages. I'm thinking possibly because the other pages have more content to them that helps spread them out more?